services/postgresflex/model_instance_create_database_request.go

```diff
@@ -11,8 +11,7 @@ API version: 1.0.0
 package postgresflex
 
 type InstanceCreateDatabaseRequest struct {
```

Comment:
breaking change

Reply:
This structs are unused, they are from endpoints that are not yet onboarded in the SDK